### DISCUSSION TITLE
Better file explorer

### DISCUSSION THREAD

**Discussion by Erevoid** [Orama-Interactive/Pixelorama#568] [Ideas]
The current way to save and browse the files from the program is a bit slow. I would suggest a file explorer that has major folders on the left, organized as a tree. A good example of this is the windows file explorer and since a lot of people are familiar with it, I believe it would be easier for everybody if this was implemented

**Comment by OverloadedOrama** [comment]
It would be very useful if Godot's FileDialog node was improved for this, as it would also help other projects. Nevertheless, we also need a way to open and save files in HTML5 and Android platforms, which may mean ditching the FileDialog node entirely, at least for these platforms.

**Comment by OverloadedOrama** [comment]
Using native file dialogs will be possible in 1.0 for Linux, Windows & macOS and can be enabled in the preferences. Implemented in 4bc0fba941944ced4483b38a665064db49375032 and d640b6a979d1cf04fd5dd7df7ae6edcfc84221ed.
